modules/geolocation/unittests.py

```diff
@@ -195,7 +195,131 @@ def test_point_set_2(self):
         # Test
         np.testing.assert_almost_equal(actual, expected)
 
+class TestConvertInput(unittest.TestCase):
+    def test_all_zeroes(self):
+
+        expected_o = np.zeros(3)
+        expected_c = np.array([1, 0 ,0])
+        expected_u = np.array([0, 1 ,0])
+        expected_v = np.array([0, 0 ,1])
+
+        latitude = 0
+        longitude = 0
+        altitude = 0
+        worldOrigin = np.array([0, 0, 0])
+        gpsOffset = np.array([0, 0, 0])
+        cameraOffset = np.array([0, 0, 0])
+        eulerCamera = {"x": 0, "y": 0, "z": 0}
+        eulerPlane = {"x": 0, "y": 0, "z": 0}
+
+
+        geoLocationClass = geolocation.Geolocation()
```

Comment:
This can be placed in the `setUp()` method (see e.g. https://github.com/UWARG/computer-vision-python/pull/26/files#diff-d96e1349803cbb8d531f917f69838c18e71873ba712c19f8977f52a4a8702a30R202).